src/styles/blocks/card.scss

```diff
@@ -0,0 +1,75 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+
+    display: block;
+    margin: 0 3px 40px;
+  }
+
+  &__title {
+    color: $main-text-color;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+
+    margin-top: 0;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    margin-bottom: 24px;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+  }
+
+  &__buy {
+    width: 100%;
+    height: 40px;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    border-radius: 5px;
+    box-sizing: border-box;
+    border: 1px solid $third-text-color;
+    background-color: $third-text-color;
+
+    color: white;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-decoration: none;
+    text-transform: uppercase;
+    transition-duration: 0.3s;
+
+    &:hover {
+      color: $third-text-color;
+      background-color: #fff;
+    }
+  }
+
+  transform: scale(1);
+  transition-duration: 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+  }
```

Comment:
The hover effect on the `.card` class scales the entire card to 1.2 times its size. This might cause layout shifts or overlap with other elements on the page. Consider using a more subtle effect or ensuring there is enough space around the card to accommodate this scaling.